src/apps/sequencer/ui/pages/NoteSequenceEditPage.cpp

```diff
@@ -384,8 +384,6 @@ void NoteSequenceEditPage::keyPress(KeyPressEvent &event) {
     }
 
     if (key.isEncoder()) {
-        setSectionTracking(false);
```

Comment:
This is what I'm talking about @aclangor , you removed this so the encoder won't stop the tracking anymore if used

Reply:
everything can be handled by page+s16. This key combo acts as a toggle: if page follow is on, it will turn it off. if page follow is off, it will turn it on.

Reply:
The point is that you want "auto off" in some cases. For example, if you hit left or right (you take over the auto mode) and, if you start inputting, hence the disabling on encoder click

Reply:
I see what you mean.

Another avenue could be to disable page follow if a step is pressed. That way it automatically stops following if you start editing a step on the page.

My only reservation to keeping it on the encoder click is that it duplicates functionality as a shortcut, where the encoder click could potentially be used for some other functionality. But there is nothing planned for it (that I'm aware of at least) so I don't have a strong argument for removing it.

Reply:
yes, i should add follow disabling too when a step is pressed, otherwise you would get strange behavior if the page changes.
About the click, it's not about using it for that. It's just a side effect of using it. Exactly like pressing is step doesn't mean you reserve it for stopping the following; it's a side effect of editing a step.

Reply:
Right now the only thing an encoder click (pressing down on the encoder) does is toggle any selected steps on or off. Was your intention to disable page follow when the encoder was turned?

Reply:
I've added the encoder click event back in for now. I do believe it is better suited for the encoder turn event when editing a step or even when clicking and holding a step, but that can be addressed later